Source: oahong@gmail.com
Upstream: No
Reason: Fix missing preprocessor directives.
--- gcin-1.4.5.pre4.orig/eve.c	2009-04-17 23:15:42.000000000 +0800
+++ gcin-1.4.5.pre4/eve.c	2009-04-18 16:44:03.000000000 +0800
@@ -1152,10 +1152,14 @@
     case 3:
     case 10:
       goto empty;
+#if USE_TSIN
     case 6:
       return tsin_get_preedit(str, attr, cursor);
+#endif
+#if USE_ANTHY
     case 12:
       return anthy_get_preedit(str, attr, cursor);
+#endif
     default:
       return gtab_get_preedit(str, attr, cursor);
 //      dbg("metho %d\n", current_CS->in_method);
@@ -1177,12 +1181,16 @@
       return;
     case 10:
       return;
+#if USE_TSIN
     case 6:
       tsin_reset();
       return;
+#endif
+#if USE_ANTHY
     case 12:
       gcin_anthy_reset();
       return;
+#endif
     default:
       gtab_reset();
 //      dbg("metho %d\n", current_CS->in_method);
